tools/interface/contract_interface.py

```diff
@@ -1,6 +1,6 @@
 # Inspired by https://github.com/pryce-turner/web3_python_tutorials
```

Comment:
This is a file I didn't use after I put black at git hooks, so it not aligned. Alignment here will change many irrelevant lines. I'll better do align it after review.